docker-compose.yaml

```diff
@@ -1,37 +1,67 @@
-version: '2.1'
-
 services:
   app:
     build:
       context: .
     working_dir: "/app"
+    platform: "linux/amd64"
```

Comment:
Why is it forcing amd64?

Reply:
I believe it's something related to the ffmpeg image, I think it compiles to use specific hardware/assembler ... If you try to remove the specific platform, the following error will show up:
```
# the compiling error will show up.
# 9.470 go: downloading golang.org/x/crypto v0.2.0
# 29.00 # runtime/cgo
# 29.00 gcc_arm64.S: Assembler messages:
# 29.00 gcc_arm64.S:30: Error: no such instruction: `stp x29,x30,[sp,'
```

If you try to use a cross platform gcc tooling `gcc-aarch64-linux-gnu g++-aarch64-linux-gnu` it will raise:

```
9.363 go: downloading golang.org/x/crypto v0.2.0
104.2 # github.com/flavioribeiro/donut
104.2 /usr/local/go/pkg/tool/linux_arm64/link: running aarch64-linux-gnu-gcc failed: exit status 1
104.2 /usr/lib/gcc-cross/aarch64-linux-gnu/12/../../../../aarch64-linux-gnu/bin/ld: skipping incompatible /usr/local/lib/libavdevice.so when searching for -lavdevice
104.2 /usr/lib/gcc-cross/aarch64-linux-gnu/12/../../../../aarch64-linux-gnu/bin/ld: skipping incompatible /usr/local/lib/libavdevice.a when searching for -lavdevice
104.2 /usr/lib/gcc-cross/aarch64-linux-gnu/12/../../../../aarch64-linux-gnu/bin/ld: skipping incompatible /usr/local/lib/libavdevice.so when searching for -lavdevice
104.2 /usr/lib/gcc-cross/aarch64-linux-gnu/12/../../../../aarch64-linux-gnu/bin/ld: skipping incompatible /usr/local/lib/libavdevice.a when searching for -lavdevice
```

I haven't tested many options on how to solve it though.